predicting_paper_venues_pt1.md code snippets to reflect updated code

**Describe the bug**
The 2nd and 4th code snippets within step 3 of ["predicting_paper_venues_pt1.md"](https://github.com/allenai/allennlp/blob/master/tutorials/getting_started/predicting_paper_venues/predicting_paper_venues_pt1.md#step-three-write-your-datasetreader) does not reflect the code that it references and if kept inconsistent will cause the error below if you use the code within the code snippets in the guide.

**To Reproduce**
1. clone the [allennlp as a library example repository](https://github.com/allenai/allennlp-as-a-library-example)
2. Change the [semantic scholar dataset reader test](https://github.com/allenai/allennlp-as-a-library-example/blob/master/tests/dataset_readers/semantic_scholar_dataset_reader_test.py) to reflect the 2nd and 4th code snippets within step 3 of  ["predicting_paper_venues_pt1.md"](https://github.com/allenai/allennlp/blob/master/tutorials/getting_started/predicting_paper_venues/predicting_paper_venues_pt1.md#step-three-write-your-datasetreader)
3. Run the pytest: `python -m pytest tests/dataset_readers/semantic_scholar_dataset_reader_test.py`

Test error produced in Additional Context.

**Expected behavior**
Pass the test

**System (please complete the following information):**
 - OS: Ubuntu 16.04
 - Python version: 3.6.7
 - AllenNLP version: 0.6.1
 - PyTorch version: 0.4.1 Cuda 9.2

**Additional context**
Test error produced:
```
tests/dataset_readers/semantic_scholar_dataset_reader_test.py F          [100%]

=================================== FAILURES ===================================
_____________ TestSemanticScholarDatasetReader.test_read_from_file _____________

self = <dataset_readers.semantic_scholar_dataset_reader_test.TestSemanticScholarDatasetReader testMethod=test_read_from_file>

    def test_read_from_file(self):
    
        reader = SemanticScholarDatasetReader()
        dataset = reader.read('tests/fixtures/s2_papers.jsonl')
    
        instance1 = {"title": ["Interferring", "Discourse", "Relations", "in", "Context"],
                     "abstract": ["We", "investigate", "various", "contextual", "effects"],
                     "venue": "ACL"}
    
        instance2 = {"title": ["GRASPER", ":", "A", "Permissive", "Planning", "Robot"],
                     "abstract": ["Execut", "ion", "of", "classical", "plans"],
                     "venue": "AI"}
    
        instance3 = {"title": ["Route", "Planning", "under", "Uncertainty", ":", "The", "Canadian",
                               "Traveller", "Problem"],
                     "abstract": ["The", "Canadian", "Traveller", "problem", "is"],
                     "venue": "AI"}
    
>       assert len(dataset.instances) == 10
E       AttributeError: 'list' object has no attribute 'instances'

tests/dataset_readers/semantic_scholar_dataset_reader_test.py:27: AttributeError
----------------------------- Captured stderr call -----------------------------
10it [00:00, 178.53it/s]
===Flaky Test Report===


===End Flaky Test Report===
=========================== 1 failed in 1.83 seconds ===========================
```

--- a/tutorials/getting_started/predicting_paper_venues/predicting_paper_venues_pt1.md
+++ b/tutorials/getting_started/predicting_paper_venues/predicting_paper_venues_pt1.md
@@ -78,12 +78,14 @@ The basic API of the `DatasetReader` is just that we need to instantiate the obj
 
 ```python
 from allennlp.common.testing import AllenNlpTestCase
+from allennlp.common.util import ensure_list
+
 from my_library.dataset_readers import SemanticScholarDatasetReader
 
 class TestSemanticScholarDatasetReader(AllenNlpTestCase):
     def test_read_from_file(self):
         reader = SemanticScholarDatasetReader()
-        dataset = reader.read('tests/fixtures/s2_papers.jsonl')
+        instances = ensure_list(reader.read('tests/fixtures/s2_papers.jsonl'))
 ```
 
 Then we just want to make sure that the resulting dataset looks like we expect.  We'll refer you to
@@ -112,16 +114,16 @@ are given as `Lists` here, but other than that, this just mimics the relevant fi
 blobs in the data file.  We'll have our test make sure that what we read matches this:
 
 ```python
-        assert len(dataset.instances) == 10
-        fields = dataset.instances[0].fields
+        assert len(instances) == 10
+        fields = instances[0].fields
         assert [t.text for t in fields["title"].tokens] == instance1["title"]
         assert [t.text for t in fields["abstract"].tokens[:5]] == instance1["abstract"]
         assert fields["label"].label == instance1["venue"]
-        fields = dataset.instances[1].fields
+        fields = instances[1].fields
         assert [t.text for t in fields["title"].tokens] == instance2["title"]
         assert [t.text for t in fields["abstract"].tokens[:5]] == instance2["abstract"]
         assert fields["label"].label == instance2["venue"]
-        fields = dataset.instances[2].fields
+        fields = instances[2].fields
         assert [t.text for t in fields["title"].tokens] == instance3["title"]
         assert [t.text for t in fields["abstract"].tokens[:5]] == instance3["abstract"]
         assert fields["label"].label == instance3["venue"]

